src/main/java/ch/njol/skript/effects/EffDropLeash.java

```diff
@@ -0,0 +1,64 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Keywords;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityUnleashEvent;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Allow / Prevent Leash Drop")
+@Description("Allows or prevents the leash to drop in an unleash event.")
+@Examples({
+	"on unleash:",
+		"\tif player is not set:",
+			"\t\tprevent the leash from dropping",
+		"\telse if player is op:",
+			"\t\tallow the leash to drop"
+})
+@Keywords("lead")
+@Events("Unleash")
+@Since("INSERT VERSION")
+public class EffDropLeash extends Effect {
+
+	static {
+		if (Skript.methodExists(EntityUnleashEvent.class, "setDropLeash", boolean.class))
```

Comment:
Should have RequiredPlugins for what versions/platforms this syntax requires to work. Note that if it was added before 1.19.4, you can remove this check

Reply:
This was added in way earlier versions than 1.19.4, so I will remove this check.